tls/s2n_record_write.c

```diff
@@ -70,6 +70,8 @@ static S2N_RESULT s2n_tls_record_overhead(struct s2n_connection *conn, uint16_t
 
 /* This function returns maximum size of plaintext data to write for the payload.
  * Record overheads are not included here.
+ *= https://tools.ietf.org/rfc/rfc8446#section-5.1
+ *# The length MUST NOT exceed 2^14 bytes.
```

Comment:
Can this comment be placed more precisely? Maybe over this? https://github.com/aws/s2n-tls/blob/458ad65cc97a6b026d89a5e762b9a84b9783dad6/tls/s2n_record_write.c#L81